lib/espnow.toit

```diff
@@ -0,0 +1,75 @@
+// Copyright (C) 2022 Toitware ApS. All rights reserved.
```

Comment:
espnow is an esp32 library.
I would suggest the following:
- create an 'esp32' folder inside the lib directory.
- move `lib/esp32.toit` into `lib/esp32`.  This way `import esp32` continues to work.
- move `lib/espnow.toit` into `lib/esp32`. This way `import esp32.espnow` works.

Reply:
OK, it's esp32's function.